cmd/synchronization/app/controller_constructor.go

```diff
@@ -79,6 +80,10 @@ func (cc *ControllerConstructor) New(config *ctrl.Config, cctx *ctrl.Context) (*
 	scHTTPClient := util.HTTPClient()
 	scClient := servicecentral.NewServiceCentralClient(config.Logger, scHTTPClient, opts.ASAPClientConfig, opts.Providers.ServiceCentralURL)
 
+	// create a client for talking to OpsGenie Integration Manager
+	ogHTTPClient := util.HTTPClient()
```

Comment:
Why are we creating a some many http clients?

Reply:
🤷‍♂️ Should I create one client and re-use for service central and opsgenie?

Reply:
As discussed outside this PR we will address this later.